### PR TITLE
adding a local copy of the turtlebot URDF to stop the need for general testing and the turtlebot repo

### PR DESCRIPTION
**Pull request by SteveMacenski**
Moved from #1438.

Basically, we don't need the turtlebot repo or workspace at all for simulation, expect for the URDF. That keeps causing me personally trouble when I have a separate workspace but more generally I think if we're going to include a world file, rviz config, launch files, etc that are self contained, we should just throw this in there as well.

Also in enabling the depth camera, we had to make some modifications to the turtlebot URDF camera link and such, so we need to have now a separate file stored somewhere. Edit: (Maybe I buried the lead on that one)

### CODE REVIEW

**Comment by mlherd** [comment]
Shouldn't the robot description files be in a directory named urdf? The world directory is used for the Gazebo world models. 
We have one urdf with depth camera stored in nav2_system_tests. Do we want to keep that one there?

**Comment by mkhansen-intel** [comment]
Just my 2 cents. We should probably put the robot and camera models in a 'models' directory and the worlds themselves in the 'worlds' directory.

**Comment by SteveMacenski** [comment]
Yeah we can stick it in a urdf directory, no problem.

**Comment by SteveMacenski** [comment]
done!

**Comment by SteveMacenski** [comment]
That is true, I mean it may make more sense to create another package `turtlebot3_bringup` that has the TB3 specific things (which is... pretty much everything). 

I think there's also a strong argument to be made that the entire `nav2_bringup` is really `turtlebot3_nav_bringup` since all but 3 launch files are for the turtlebot3 (urdf, model, world, maps, configs, ...)

**Comment by mlherd** [comment]
For now these changes make sense to eliminte the turtlebot3 pacakage dependency for simulation, but all the tb3 and gazebo files in the repo need to be better orginized in the future.

File: nav2_bringup/bringup/launch/nav2_tb3_simulation_launch.py [hunk 1]
```diff
@@ -140,8 +140,7 @@ def generate_launch_description():
         cmd=['gzclient'],
         cwd=[launch_dir], output='screen')
 
-    urdf = os.path.join(
-        get_package_share_directory('turtlebot3_description'), 'urdf', 'turtlebot3_waffle.urdf')
+    urdf = os.path.join(bringup_dir, 'urdf', 'turtlebot3_waffle.urdf')
```
Comment (mlherd) [inline nav2_bringup/bringup/launch/nav2_tb3_simulation_launch.py]: file name ``turtlebot_waffle.urdf`` needs to match with the file name in the launch file.

Comment (SteveMacenski) [reply nav2_bringup/bringup/launch/nav2_tb3_simulation_launch.py]: Some PRs man.. you change something, you break 2 more things ;-)

**Pull request merged**